testdata/foundry.toml

```diff
@@ -0,0 +1,45 @@
+[default]
```

Comment:
is this needed?

Reply:
the file itself? no. i am actually a bit annoyed by the structure of testdata directory personally. I think it should be a standard forge style structure. interested in your thoughts @onbjerg